README.txt

```diff
@@ -439,7 +439,7 @@ Solaris 11:
 
 Debian/Ubuntu:
 
-  # apt-get install sun-java6-javadb
+  # apt-get install sun-java8-javadb
 
```

Comment:
only in jessie - https://packages.debian.org/jessie/libderby-java
jessie will be out this week(next week) so I will probably use that?